packages/experimental/Badge/src/CounterBadge/CounterBadge.types.ts

```diff
@@ -0,0 +1,71 @@
+import { BadgeCoreProps, BadgeSlotProps } from '../';
+import { BadgeTokens, BadgeProps } from '../Badge.types';
+
+export const counterBadgeName = 'CounterBadge';
+
+export interface CounterBadgeProps extends Omit<BadgeCoreProps, 'appearance' | 'color'>, Omit<BadgeProps, 'shape'> {
+  /**
+   * A Badge can have different appearances that emphasize certain parts of it:
+   *  - filled: The default appearance if one is not specified.
+   *    The badge background is filled with color with a contrasting foreground text to match.
+   *  - ghost: The badge background is transparent, with the foreground text taking color to emphasize it.
+   * @default filled
+   */
+  appearance?: 'filled' | 'ghost';
+
+  /**
+   * Semantic colors for a counter badge
+   * @default brand
+   */
+  color?: 'brand' | 'danger' | 'important' | 'informative';
+
+  /**
+   * Value displayed by the Badge
+   * @default 0
+   */
+  count?: number;
+
+  /**
+   * If a dot should be displayed without the count
+   * @default false
+   */
+  dot?: boolean;
```

Comment:
This behavior doesn't seem to be implemented, did I miss it? It seems like the displayCount from the state is always passed through to the Text slot regardless of dot?

Reply:
Nvm I see you'll implement it later